crowd_anki/utils/pathlib_wrapper.py

```diff
@@ -0,0 +1,4 @@
+try:
+    from pathlib import Path
+except ImportError:
+    from ..thirdparty.pathlib import Path
```

Comment:
thirdparty.pathlib is itself a wrapper, I'm not sure I understood why you can't reuse it, but if you can't, you might as well delete it and do aliasing that it performs directly here